v2_papers/2009/kind_histogram.json

```diff
@@ -0,0 +1,179 @@
+        "types": [],
+        "generation": "altered_captured",
+        "generation_year": 1999,
+        "covered_period": "weeks",
+        "details": [
+          "raw"
+        ],
+        "subsets": []
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": true,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [],
+    "normalization_type": "no",
+    "transformations": [
+      "_histograms",
+      "_metric_space",
+      "entropy"
+    ],
+    "final_data_format": "_histograms",
+    "feature_selections": [
+      {
+        "name": "constant-removal",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      },
+      {
+        "name": "pca",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      }
+    ],
+    "flows": [
+      {
+        "selection": "study_based",
+        "role": "main",
+        "main_goal": "anomaly_detection",
+        "active_timeout": "missing",
+        "idle_timeout": "missing",
+        "features": [
+          "sourceIPv4Address",
+          "destinationIPv4Address",
+          "sourceTransportPort",
+          "destinationTransportPort",
+          "protocolIdentifier",
+          "octetTotalCount",
+          "packetTotalCount",
+          "flowDurationMilliseconds"
+        ],
+        "key_features": []
+      }
+    ]
+  },
+  "analysis_method": {
+    "supervised_learning": true,
+    "unsupervised_learning": true,
+    "semisupervised_learning": false,
+    "anomaly_detection": true,
+    "tools": [],
+    "algorithms": [
+      {
+        "name": "hierarchical cluster analysis",
+        "subname": "hca",
+        "learning": "unsupervised",
+        "role": "intermediate",
+        "type": "clustering",
+        "metric/decision_criteria": "euclidean",
+        "source": "referenced",
+        "parameters_provided": false
+      },
+      {
+        "name": "clustering",
+        "subname": "K-means",
+        "learning": "unsupervised",
+        "role": "intermediate",
+        "type": "clustering",
+        "metric/decision_criteria": "euclidean",
+        "source": "referenced",
+        "parameters_provided": false
+      }
+    ]
+  },
+  "evaluation": {
+    "algorithm_comparison": false,
+    "internal_validation": true,
+    "external_validation": true,
+    "dpi-based_validation": false,
+    "port-based_validation": false,
+    "pre-knowledge-based_validation": false,
+    "manual_verification": false,
+    "implementation_in_real_scenario": false,
+    "train_test_separation": true,
+    "methods": [
+      {
+        "name": "normal classification metrics",
+        "type": "external",
+        "metrics": [
+          "accuracy"
+        ],
+        "source": "popular"
+      }
+    ]
+  },
+  "result": {
+    "main_goal": "anomaly_detection",
+    "focus_main": "_new_approach",
```

Comment:
I'd say the main focus is the methodology, so the ``methodology/framework`` value.

Reply:
The goal from the beginning was to detect anomalies, they presented a way to represent the data via histograms and then detect abnormal ones. What do you think ?